test/fixtures/dialects/snowflake/jinja_tricky_whitespace_control.sql

```diff
@@ -0,0 +1,88 @@
+-- Added for issue 2786. This file originally failed to parse due to invalid slice
+-- output from JinjaTracer.
+{{
+    config(
+        materialized='incremental',
+        unique_key='md5_surrogate_key_main'
+    )
+}}
```

Comment:
Is this not DBT? So can't be in the non-dbt fixtures directory?

Reply:
IIUC, the Jinja templater has a handful of dbt-like macros built in. I was actually surprised myself that this worked.

Reply:
I do need to move it, because even Jinja is not supported in this directory.